src/classes/PromoModals.js

```diff
@@ -40,16 +42,18 @@ class PromoModals {
 	 */
 	static whichPromoModalShouldWeDisplay() {
```

Comment:
Are we never displaying Premium modals anymore, then?

Reply:
That's the plan, we're also keeping the Premium modal code in there in case we want to switch back at some point